Gemfile.lock

```diff
@@ -0,0 +1,211 @@
+GEM
```

Comment:
Do you think we should include `Gemfile.lock`, or maybe all `.lock` file in `.gitignore`?

Reply:
I think keeping the `.lock` files under source control is convention, since the `.lock` files contain the specific versions of the dependencies that are installed to make the tool work. If we remove the .lock file, then if in the future one of our dependencies gets updated in a way that breaks our code, new developers on the project would be out of luck.